guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -0,0 +1,365 @@
+
+[[Content_View]]
+Content View:: A subset of Library content created by intelligent filtering.
+Once a content view is published, it can be promoted through the lifecycle environment path, or modified using incremental upgrades.
+
+[[Discovered_Host]]
+Discovered Host:: A bare-metal host detected on the provisioning network by the Discovery plugin.
+
+[[Discovery_Image]]
+Discovery Image:: Refers to the minimal operating system based on {EL} that is PXE-booted on hosts to acquire initial hardware information and to communicate with {ProjectServer} before starting the provisioning process.
+
+[[Discovery_plugin]]
+Discovery plugin:: Enables automatic bare-metal discovery of unknown hosts on the provisioning network.
+The plugin consists of three components: services running on {ProjectServer} and {SmartProxyServer}, and the Discovery image running on host.
+
+[[Discovery_Rule]]
+Discovery Rule:: A set of predefined provisioning rules which assigns a host group to discovered hosts and triggers provisioning automatically.
+
+[[Docker_Tag]]
+Docker Tag:: A mark used to differentiate container images, typically by the version of the application stored in the image.
+In the {ProjectWebUI}, you can filter images by tag under *Content* > *Docker Tags*.
+
+ifndef::satellite[]
+[[Enterprise_Linux]]
+Enterprise Linux:: An umbrella term for the following {RHEL}-like operating systems:
+
+ AlmaLinux
+ CentOS Linux
+ CentOS Stream
+ Oracle Linux
+ {RHEL}
+ Rocky Linux
+
+ifndef::orcharhino[]
+{Project} is tested on AlmaLinux and CentOS Stream.
+endif::[]
+endif::[]
+
+[[ERB]]
+ERB:: Embedded Ruby (ERB) is a template syntax used in provisioning and job templates.
+
+[[Errata]]
+Errata:: Updated RPM packages containing security fixes, bug fixes, and enhancements.
+In relationship to a host, erratum is *applicable* if it updates a package installed on the host and *installable* if it is present in the host's content view (which means it is accessible for installation on the host).
+
+[[External_Node_Classifier]]
+External Node Classifier:: A construct that provides additional data for a server to use when configuring hosts.
+{ProjectName} acts as an External Node Classifier to Puppet servers in a {Project} deployment.
++
+Note that the External Node Classifier will be removed in the next {Project} version.
+
+[[Facter]]
+Facter:: A program that provides information (facts) about the system on which it is run; for example, Facter can report total memory, operating system version, architecture, and more.
+Puppet modules enable specific configurations based on host data gathered by Facter.
+
+[[Facts]]
+Facts:: Host parameters such as total memory, operating system version, or architecture.
+Facts are reported by Facter and used by Puppet.
+
+[[Foreman]]
+Foreman:: The component mainly responsible for provisioning and content lifecycle management.
+ifdef::satellite[]
+Foreman is the main upstream counterpart of {ProjectName}.
+endif::[]
+
```

Comment:
```suggestion
```
This definition of Foreman is not sufficiently detailed and thus won't be particularly helpful to users. If users don't know what Foreman is, glossary is not the place to search for more details (but rather other documentation, like the Planning guide).